### ISSUE TITLE
problems when rbinding estimates/estimands with different number of columns

### ISSUE THREAD

**Issue by acoppock** [DeclareDesign/DeclareDesign#42]
in `declare_design()` we do things like `rbind(estimates_df, causal_order[[i]](current_df))` which breaks if `estimates_df` and ` causal_order[[i]](current_df))` have different numbers of columns.

**Comment by graemeblair** [comment]
still broken: 

```
library(DeclareDesign)

# M -- Model: Speculation on variables and relations between them
population <- declare_population(N = 100, u = rnorm(N))
potential_outcomes <- declare_potential_outcomes(Y_Z_0 = 0, 
                                                 Y_Z_1 = 1 + u)

# D -- Data Strategy: Researcher interventions on the world
assignment <- declare_assignment(m = 50) 

# A -- Answer Strategy: Conclusions to be drawn from data
estimator <- declare_estimator(Y ~ Z, estimand = estimand)
estimator2 <- declare_estimator(handler = function(data)
  return(data.frame(est = 5, se = 3)), label = "second_estimator")

# Design: Putting it all together
design <- declare_design(population, 
                         potential_outcomes, 
                         assignment, 
                         reveal_outcomes, 
                         estimator, estimator2)
```

**Comment by nfultz** [comment]
Fixed by #150
